app/_posts/2024-06-25-kuma-2-8-0.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kuma 2.8 release with MeshExternalService, MeshPassthrough and more...
+description: Kuma 2.8 release with MeshExternalService, MeshPassthrough and more...
+date: 2024-06-25
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.8, featuring new and exciting capabilities such as `MeshExternalService`, `MeshPassthrough` policy, along with numerous improvements and groundwork for future developments.
+
+# Notable features
+* A new `MeshExternalService` resource that overcomes existing limitations with the `ExternalService` resource.
+* A new `MeshPassthrough` policy that allows exposing external endpoints for rich clients through the mesh and support for wildcard domains.
+* HostnameGenerator provides a way to generate custom domains for your `MeshExternalServices`, offering enhanced security.
```

Comment:
This sounds a bit weird 🤔 (the second part) and what security benefits does it offer?

```suggestion
* HostnameGenerator provides a way to generate custom domains for your `MeshExternalServices`.
```